data/db-schema/src/main/resources/net/corda/db/schema/crypto/migration/crypto-creation-v1.0.xml

```diff
@@ -2,8 +2,15 @@
 <databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.3.xsd">
+    <property name="schema.name" value="CRYPTO" global="false"/>
+
     <changeSet author="R3.Corda" id="crypto-creation-v1.0">
-        <createTable tableName="crypto_wrapping_key">
+
+        <sql>
+            CREATE SCHEMA IF NOT EXISTS CRYPTO;
+        </sql>
```

Comment:
Ideally this shouldn't be in here as the schema should be created outside of this. I thought the cluster bootstrapper already did that?
We can leave this in, it'll just mean we have to take it out later on.

Reply:
it has be deleted